README.md

```diff
@@ -0,0 +1,41 @@
+## Screens
+
+### Development
+
+#### Prerequisites
+- [Docker](https://www.docker.com/docker-toolbox)
+- [Heroku Toolbelt](https://toolbelt.heroku.com/)
+- Heroku Docker plugin -- `heroku plugins:install heroku-docker`
+
+#### Setting up development environment
+- Clone the repository -- `git clone git@github.com:ftlabs/screens.git`
+- Change in repository directory -- `cd screens`
+- If running OS X or Windows, [follow these steps for creating a virtual machine for Docker](#creating-a-virtual-machine-for-docker) -- This is a section in this file.
+- Build a Docker image -- `docker build .`
+- Spin up the web process in a container -- `docker-compose up`
+- Open the application in your browser of choice -- `open "http://$(docker-machine ip default):8080"`
+
+##### [Creating a virtual machine for Docker](#creating-a-virtual-machine-for-docker)
+ [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMWare](http://www.vmware.com/uk/).
```

Comment:
so, do I need to read these? Or just one? Which one?
Or do the steps below cover enough detail for me to do it without reading ether/both the pages above?